app/scripts/lib/marketing-email-client.js

```diff
@@ -9,7 +9,8 @@
 define(function (require, exports, module) {
```

Comment:
While we are here, do you wanna nuke this `define` wrapper? :D

Reply:
I was tempted to, but didn't want to cause a blowup in the diff size. Will remove the wrapper now. ;)